src/SFA.DAS.AANHub.Api/Models/CreateAdminModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SFA.DAS.AANHub.Application.Admins.Commands;
+
+namespace SFA.DAS.AANHub.Api.Models
+{
+    public class CreateAdminModel
+    {
+        public string UserName { get; set; } = null!;
+        public string Email { get; set; } = null!;
+        public string Name { get; set; } = null!;
+        public string Information { get; set; } = null!;
+        public DateTime Joined { get; set; }
+        public List<int>? Regions { get; set; }
+
+        public static implicit operator CreateAdminMemberCommand(CreateAdminModel model) => new()
```

Comment:
Add coverage for the implicit operator.  Here's an example from roatp if it helps: https://github.com/SkillsFundingAgency/das-roatp-api/blob/main/src/SFA.DAS.Roatp.Application.UnitTests/Courses/GetProviderDetailsForCourse/ProviderDetailsTests.cs